### PR TITLE
gh-issues: change the title of about page

### PR DESCRIPTION
**Pull request by adrien5054**
* In the about page, the title of the tab : 'Home' --> 'About'

### CODE REVIEW

File: ui/src/routes/about_us.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ function AboutUs() {
   return (
     <Layout className="layout">
       <Helmet>
-        <meta name="description" content={HOME_PAGE_METATAG_CONTENT} />
```
Comment (drjova) [inline ui/src/routes/about_us.tsx:47]: we shouldn't remove the description.

File: backend/docker-compose.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 version: "2.3"
 services:
   db:
-    image: postgres:9.6
+    image: postgres:14.8
```
Comment (drjova) [inline backend/docker-compose.yml:4]: why do we need this change?

**Review by drjova — COMMENTED** [review]
@adrien5054 could you please also update the github message to:

`routes: change the title of the about page`

File: ui/src/routes/about_us.tsx (hunk 1 above)
Comment (adrien5054) [reply ui/src/routes/about_us.tsx:47]: I change the tag `meta` by `title` in the other pr

Comment (drjova) [reply ui/src/routes/about_us.tsx:47]: hmm what do you mean? the other PR is for the picture. We shouldn't remove the `description`

Comment (adrien5054) [reply ui/src/routes/about_us.tsx:47]: Oh sorry I thought I was on the other pr, forgot what I said

**Review by drjova — APPROVED** [review]

**Pull request merged**